### DISCUSSION TITLE
Runpod 3090 slow it/s?

### DISCUSSION THREAD

**Discussion by meanin2** [TheLastBen/fast-stable-diffusion#2570] [Q&A] [answered]
I'm generating images with a 3090 on runpod, using automatic1111 with SDXL. (thank you @TheLastBen for making this so easy!)

It seems like the runpod image generation with sdxl (1024x1024) seems to be on the slower side with each image taking about 1 minute to generate. 
In comparison on my local setup it also takes about 1 min per picture. (Predator Helios 300, 64gb ram, 3070 ti 8gb, laptop).

Does this make sense, am i doing something wrong? 
Is there anything I can do to make runpod generate faster?

Thanks!

**Comment by TheLastBen** [comment, marked as answer]
with 3090, you should generate in under a few seconds, try a new pod, some pods have issues with their GPUs
